terraform/helm/fullnode/values.yaml

```diff
@@ -7,6 +7,9 @@ aptos_chains:
   devnet:
     waypoint_txt_url: https://devnet.aptoslabs.com/waypoint.txt
```

Comment:
nit: shall we also change this to reference `aptos-labs/aptos-genesis-waypoint` repo?

Reply:
yeah but it's currently not up to date I think